studio/schemas/documents/page.js

```diff
@@ -11,14 +11,29 @@ export default {
     {
       name: 'slug',
       type: 'slug',
+      title: 'Slug',
       required: true,
       options: {
-        source: (doc, options) => options.parent.title,
+        source: 'title',
       },
     },
     {
       name: 'content',
-      type: 'pageContent',
+      type: 'pageText',
     },
   ],
+  preview: {
```

Comment:
Maybe we should use this for all the different pages?

Reply:
I think yes